lnd.go

```diff
@@ -1011,6 +1020,7 @@ func waitForWalletPassword(
 		walletInitParams := &WalletUnlockParams{
 			Password:       unlockMsg.Passphrase,
 			RecoveryWindow: unlockMsg.RecoveryWindow,
+			Wallet:         unlockMsg.Wallet,
```

Comment:
can we also avoid unloading the wallet in `case initMsg := <-pwService.InitMsgs:`?

Reply:
Makes a lot of sense! Fixed it.